lib/nx_signal/windows.ex

```diff
@@ -289,6 +290,81 @@ defmodule NxSignal.Windows do
     end
   end
 
+  @doc """
+  Creates a Kaiser window of size `window_length`.
+
+  The Kaiser window is a taper formed by using a Bessel function.
+
+  ## Options
+
+    * `:n` - The window length. Mandatory option.
+    * `:is_periodic` - If `true`, produces a periodic window,
+       otherwise produces a symmetric window. Defaults to `true`
+    * `:type` - the output type for the window. Defaults to `{:f, 32}`
+    * `:beta` - Shape parameter for the window. As beta increases, the window becomes more focused in frequency domain. Defaults to 12.0.
+    * `:name` - the axis name. Defaults to `nil`
+
+  ## Examples
+      iex> NxSignal.Windows.kaiser(n: 4, beta: 12.0, is_periodic: true)
+      #Nx.Tensor<
+        f32[4]
+        [5.2776191296288744e-5, 0.21566684544086456, 1.0, 0.21566684544086456]
+      >
+
+      iex> NxSignal.Windows.kaiser(n: 5, beta: 12.0, is_periodic: true)
+      #Nx.Tensor<
+        f32[5]
+        [5.2776191296288744e-5, 0.10171464085578918, 0.7929376363754272, 0.7929376363754272, 0.10171464085578918]
+      >
+
+      iex> NxSignal.Windows.kaiser(n: 4, beta: 12.0, is_periodic: false)
+      #Nx.Tensor<
+        f32[4]
+        [5.2776191296288744e-5, 0.5188400149345398, 0.5188400149345398, 5.2776191296288744e-5]
+      >
+  """
+  @doc type: :windowing
+  defn kaiser(opts \\ []) do
+    opts = keyword!(opts, [:n, :name, beta: 12.0, is_periodic: true, type: {:f, 32}])
+    {l, opts} = pop_window_size(opts)
+    name = opts[:name]
+    type = opts[:type]
+    beta = opts[:beta]
+    is_periodic = opts[:is_periodic]
+
+    window_length = if is_periodic, do: l + 1, else: l
+    alpha = (window_length - 1) / 2.0
+
+    n = Nx.iota({window_length}, names: [name], type: type)
+    ratio = (n - alpha) / alpha
```

Comment:
```suggestion
    ratio = Nx.linspace(-1, 1, n: window_length, endpoint: true, type: type) |> Nx.rename([name])
```

linspace does accept a name option, but there seems to be a bug there. we can use Nx.rename instead.

```
iex(2)> window_length = 10
10
iex(3)> alpha = (window_length - 1) / 2
4.5
iex(4)> Mix.install [:nx]
:ok
iex(5)> Nx.iota({window_length}) |> Nx.subtract(alpha) |> Nx.divide(alpha)
#Nx.Tensor<
  f32[10]
  [-1.0, -0.7777777910232544, -0.5555555820465088, -0.3333333432674408, -0.1111111119389534, 0.1111111119389534, 0.3333333432674408, 0.5555555820465088, 0.7777777910232544, 1.0]
>
iex(6)> Nx.linspace(-1, 1, n: window_length, endpoint: true)
#Nx.Tensor<
  f32[10]
  [-1.0, -0.7777777910232544, -0.5555555820465088, -0.3333333134651184, -0.1111111044883728, 0.11111116409301758, 0.3333333730697632, 0.5555555820465088, 0.7777777910232544, 1.0]
>
```